content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -7,101 +7,81 @@ min-kubernetes-server-version: 1.22
+- When you need to perform tasks which require the networking namespace of the host.
 HostProcess containers have access to the host's network interfaces and IP addresses.
 - You need access to resources on the host such as the filesystem, event logs, etc.
 - Installation of specific device drivers or Windows services.
-- Consolidation of administrative tasks and security policies. This reduces the degree of 
+- Consolidation of administrative tasks and security policies. This reduces the degree of
 privileges needed by Windows nodes.
 
 
 ## {{% heading "prerequisites" %}}
 
 {{% version-check %}}
 
-To enable HostProcess containers while in Alpha you need to pass the following feature gate flag to 
-**kubelet** and **kube-apiserver**. 
-See [Features Gates](/docs/reference/command-line-tools-reference/feature-gates/#overview) 
-documentation for more details.
+To enable HostProcess containers while you need to pass the following feature gate flag to 
+**kubelet** and **kube-apiserver**:
 
-```
+```powershell
 --feature-gates=WindowsHostProcessContainers=true
 ```
 
-You can use the latest version of Containerd (v1.5.4+) with the following settings using the containerd 
-v2 configuration. Add these annotations to any runtime configurations were you wish to enable the 
-HostProcess container feature.
-
-
-```
-[plugins]
-  [plugins."io.containerd.grpc.v1.cri"]
-    [plugins."io.containerd.grpc.v1.cri".containerd]
-      [plugins."io.containerd.grpc.v1.cri".containerd.default_runtime]
-        container_annotations = ["microsoft.com/hostprocess-container"]
-        pod_annotations = ["microsoft.com/hostprocess-container"]
-    [plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
-      [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runhcs-wcow-process]
-        container_annotations = ["microsoft.com/hostprocess-container"]
-        pod_annotations = ["microsoft.com/hostprocess-container"]
-```
+See [Features Gates](/docs/reference/command-line-tools-reference/feature-gates/#overview)
+documentation for more details.
 
-The current versions of containerd ship with a version of hcsshim that does not have support. 
-You will need to build a version of hcsshim from the main branch following the 
-[instructions in hcsshim](https://github.com/Microsoft/hcsshim/#containerd-shim). 
-Once the containerd shim is built you can replace the file in your contianerd installation. 
-For example if you followed the instructions to 
-[install containerd](/docs/setup/production-environment/container-runtimes/#containerd)
-replace the `containerd-shim-runhcs-v1.exe` is installed at `$Env:ProgramFiles\containerd` with the newly built shim.
+You can use the latest version of containerd (v1.6+) to run HostProcess containers. The kublet
+communicates with containerd directly by passing the hostprocess flag via CRI.
+In 1.23 this means that the annotation plugin configurations are no longer needed.
```

Comment:
Should we drop the comment about configuring annotations since we are recommending folks just use containerd v1.6?

Reply:
This is only if we update the 1.22 docs right?

Reply:
We should probably remove it from the v1.23 docs either way.
The annotations do not work with v1.23 so i think maybe just not mention them?

Reply:
+1 not mentioning them here to avoid any confusion. Majority of the folks landing on the 1.23 docs won't have any of the history related to annotations (although I believe they will work if containerd is configured to pass them through). 

Do we need to have a separate update for 1.22 docs to remove them once 1.6 is officially out?

Reply:
I'll create a PR into the 1.22 docs.